src/executor.cpp

```diff
@@ -1064,18 +1068,19 @@ types::ReturnStatus
             case 70:
                 sendFuncNumToPhyp(funcNum);
                 break;
-
-            default:
-                std::cerr << "Given function number "
-                          << static_cast<int>(funcNum) << " is not supported."
-                          << std::endl;
-                executionStatus = std::make_tuple(false, "", "");
         }
     }
+    catch (const sdbusplus::exception_t& e)
```

Comment:
Isn't this derived from std::exception? If so it can be caught there as well.
When the code is same in both the cases, base catch is sufficient.